pkg/network/dhcp_daemon.go

```diff
@@ -6,7 +6,59 @@ import (
 	"log"
 )
 
-// UseDHCP determines if the the DHCP CNI plugin running as a daemon should be rendered.
+// UseDHCPRaw determines if the the DHCP CNI plugin running as a daemon should be rendered.
+func UseDHCPRaw(addnet *operv1.AdditionalNetworkDefinition) bool {
```

Comment:
Hmm. Do we need to look in to RawCNI like this? Is this to support 4.1 clusters?

Reply:
Nevermind, I see this is pulled from another location.